component.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/moqui-conf-2.1.xsd"
+           name="mantle-shipstation" version="1.0.0">
+    <depends-on name="mantle"/>
+    <depends-on name="oms"/>
```

Comment:
No need to add both component ad dependent, include only oms, as oms id depends on mantle